app/bundles/CampaignBundle/Translations/en_US/flashes.ini

```
@@ -5,3 +5,11 @@ mautic.campaign.error.edit_events.request_invalid="contactId and/or eventId are
 mautic.campaign.error.decision_cannot_be_scheduled="A decision type event cannot be scheduled. Event: %event%, campaign: %campaign%, contact: %contact%."
 mautic.campaign.error.notfound="No campaign with an id of %id% was found!"
 mautic.campaign.notice.batch_deleted="%count% campaigns have been deleted!"
+mautic.campaign.notice.import.finished="'%name%' (ID: %id%) has been successfully imported."
+mautic.campaign.error.import.file.missing="File '%file%' cannot be found in the filesystem. Please try again."
+mautic.campaign.error.export.file_not_found="Export file could not be created: %path%"
+mautic.campaign.error.export.no_campaigns_selected="No campaigns found for export.",
```

```suggestion
mautic.campaign.error.export.no_campaigns_selected="No campaigns found for export."
```
The comma is still here

---

fixed